polars/polars-arrow/src/kernels/rolling/no_nulls/min_max.rs

```diff
@@ -18,252 +18,164 @@ impl<'a, T: NativeType> RollingAggWindowNoNulls<'a, T> for SortedMinMax<'a, T> {
     }
 }
 
+#[inline]
+unsafe fn get_min_and_ix<T>(slice: &[T], start: usize, end: usize) -> Option<(usize, &T)>
+where
+    T: NativeType + IsFloat + PartialOrd,
+{
+    slice
+        .get_unchecked(start..end)
+        .iter()
+        .enumerate()
+        .rev()
```

Comment:
So we iterate in reverse to ensure we get the latest index? If so, could you add a comment about that in this function.